support/Procfile

```diff
@@ -1,5 +1,5 @@
-redis: redis-server
-api: target/debug/bldr-api start
+redis: redis-server --dir tmp
```

Comment:
@reset Yeah, I could do that--I was starting to wonder if the file data persisted between invocations, why not the redis data (if available)? More of a copying Rails pattern kind of thing.